Library/Formula/boxes.rb

```diff
@@ -17,7 +17,7 @@ def install
 
     # distro uses /usr/share/boxes change to prefix
     system "make",
-      "GLOBALCONF=#{share}/boxes-config",
+      "GLOBALCONF=#{pkgshare}/boxes-config",
```

Comment:
I think you might have discovered an edge case in my `audit` check here :wink:. This is actually installing as:

```
HOMEBREW_CELLAR/boxes/version/share/boxes-config
```
Rather than:
```
HOMEBREW_CELLAR/boxes/version/share/boxes/boxes-config
```
So it's fine not to  use `pkgshare`. I'm not necessarily opposed though; It is a bit weird to have a `boxes-config` file dumped in `HOMEBREW_PREFIX/share`.